--- a/src/warnet/control.py
+++ b/src/warnet/control.py
@@ -308,6 +308,8 @@ def _run(
                 "commander.py",
                 "test_framework",
                 "ln_framework",
+                "lib/",
+                "config/",
                 scenario_path.name,
             ]
         ):

--- a/src/warnet/control.py
+++ b/src/warnet/control.py
@@ -438,1 +438,1 @@
-    wait_for_init(name, namespace=namespace)
+    wait_for_init(name, namespace=namespace, timeout=600)
